src/app/config-handler.ts

```diff
@@ -324,7 +324,9 @@ class Config {
             await app.whenReady();
             await this.readGlobalConfig();
             logger.info(`config-handler: user config doesn't exist! will create new one and update config`);
-            await this.updateUserConfig({ configVersion: app.getVersion().toString(), buildNumber, ...this.globalConfig } as IConfig);
+            const newUserConfig = this.globalConfig as IConfig;
+            delete newUserConfig.url;
```

Comment:
This will also remove the URL field from `this.globalConfig`

Reply:
You are right, fixed and pushed a new commit.